dato.config.js

```diff
@@ -453,7 +453,7 @@ function getPages (dato, chapterRef) {
   return pages
     .filter(filterPublished)
     .map(page => {
-      const { body, files, graphs, images, keywords, slug, title, video, mapboxStyle } = page;
+      const { body, files, graphs, images, keywords, slug, title, video, videoChina, mapboxStyle } = page;
```

Comment:
If you do this, I think you don't have to make any changes to the components except add a `v-if="videoProvider === 'qq'"` in `ResponsiveVideo.vue`:

```js
let video = page.video
if (page.videoChina) {
  const videoProviderUid = ^https:\/\/v\.qq\.com\/x\/page\/([a-z0-9]+)\.html$.exec(page.videoChina);
  video = {
      videoProvider: 'qq',
      videoProviderUid,
  };
}
```

Reply:
Yes, will do!